gitapp/lib/user_details/user_details.dart

```diff
@@ -13,9 +13,73 @@ class DetailScreen extends StatelessWidget {
       appBar: AppBar(
         title: Text("User details"),
       ),
-      body: Padding(
-        padding: EdgeInsets.all(16.0),
-        child: Text("User details screen"),
+      body: Container(
+        padding: EdgeInsets.all(30.0),
+        child: Row(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            mainAxisAlignment: MainAxisAlignment.start,
+            children: <Widget> [
+              Container(
+
+                height: 150.0,
+                width: 150.0,
+                decoration: BoxDecoration(
+                  shape: BoxShape.circle,
+                  border: Border.all(width: 3.0, color: Colors.white),
```

Comment:
if you are using white colour, then still please move it to constants. In this way we will have all the colours in one place and under control.

Reply:
Solved